tests/test_notify_celery.py

```diff
@@ -0,0 +1,86 @@
+import uuid
```

Comment:
We should move these test files to match the new directory structure.

Reply:
Sorry, I could have been more specific: normally we name the test files the same as the code files, so it's clear where the code they're testing is (and avoids making unnecessary decisions about file names).

So in this case it would be:

- `tests/app/celery/test_celery.py`
- `tests/app/celery/test_tasks.py`

I can see [FTP has gone its own way here](https://github.com/alphagov/notifications-ftp/tree/master/tests/app/celery), but you can see the convention in other apps like [API](https://github.com/alphagov/notifications-api/tree/master/tests/app/celery) and [Template Preview](https://github.com/alphagov/notifications-template-preview/tree/master/tests/celery).

Reply:
Force pushed here: https://github.com/alphagov/notifications-govuk-alerts/compare/b03419145131b6343f6c9455c433f5a1c19c99d0..6e8cce1a58908ecbe88cf8962586012d6e343042